components/expenses/PayoutBankInformationForm.js

```diff
@@ -45,7 +45,7 @@ const FormFieldsContainer = styled(Flex)`
 `;
 
 const requiredFieldsQuery = gqlV2`
-  query Collective($slug: String, $currency: TransferWiseCurrency!) {
+  query Collective($slug: String, $currency: String!) {
```

Comment:
Is this change intended?

Reply:
Yes, otherwise any currency update in TransferWise will cause errors on our side.